src/providers/DataProvider.ts

```diff
@@ -131,6 +231,10 @@ export class DataProvider {
     resource: string,
     params: GetOneParams
   ): Promise<GetOneResult<RecordType>> => {
+    if (resource === "users") {
```

Comment:
I probably should change "users" by "cognitoUsers" or something like that...

Reply:
I agree, I think `users` is too common, `cognitoUsers` seems fine